Userland/Libraries/LibELF/DynamicLinker.cpp

```diff
@@ -108,7 +110,22 @@ static Result<NonnullRefPtr<DynamicLoader>, DlErrorMessage> map_library(const St
         return map_library(name, fd);
     }
 
-    // TODO: Do we want to also look for libs in other paths too?
+    // Scan the LD_LIBRARY_PATH envvar if it exists or we're not a SUID program
+    if (s_ld_library_path != nullptr) {
+        auto paths = StringView { s_ld_library_path }.split_view(':');
+        for (const auto& search_path : paths) {
+            StringBuilder path_builder;
+            path_builder.append(search_path);
+            path_builder.append(name);
+            int fd = open(path_builder.to_string().characters(), O_RDONLY);
+            if (fd < 0)
+                continue;
+            return map_library(name, fd);
+        }
+    }
+
+    // Now check the default paths. The manpage for `dlopen()` state we should
+    // check /lib and then `/usr/lib`, however we don't have a `/lib` directory yet.
```

Comment:
Those are linux's definitions, Dr. POSIX says ["do whatever the fuck you want"](https://pubs.opengroup.org/onlinepubs/9699919799/functions/dlopen.html):
> _file_ is used in an implementation-defined manner to yield a pathname

Reply:
Trolled yet again, it would seem!